test/Swashbuckle.AspNetCore.SwaggerGen.Test/snapshots/VerifyTests.GenerateSchema_PreservesIntermediateBaseProperties_WhenUsingOneOfPolymorphism.DotNet10_0.verified.txt

```diff
@@ -0,0 +1,62 @@
+﻿{
+  "openapi": "3.0.4",
+  "info": {
+    "title": "Test API",
+    "version": "V1"
+  },
+  "paths": {
+    "/resource": {
+      "post": {
+        "tags": [
+          "Fake"
+        ],
+        "requestBody": {
+          "content": { }
+        },
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    }
+  },
```

Comment:
Could you add a new controller with an action that uses the type? Otherwise it looks a bit odd as the types are "just there", rather than there because a path uses them. That way it's a bit more realistic.

Otherwise the schemas _look_ correct, and some of the other validation we have should flush out any problems if there's any.